src/constructorio-id.js

```diff
@@ -103,28 +113,58 @@
       try {
         data = JSON.parse(localStorage.getItem(key));
       } catch (e) {
-        // fail silently
+        if (
```

Comment:
This function does not seem correct anymore?

Reply:
It got more difficult since client-id's are strings while session-id's are objects, and these functions (get, set) now need to handle both. Any suggestions now to make easier to understand or improve?